starter-code/src/app/components/my-home/my-home.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
+import { MoviesService } from '../../services/movies.service';
+
+@Component({
+  selector: 'app-my-home',
+  templateUrl: './my-home.component.html',
+  styleUrls: ['./my-home.component.css'],
+  providers: [MoviesService]
```

Comment:
This works but since you are adding the service as a provider in the app.module.ts file you do not need to add it here